ecosystem/html_templates/style.css.jinja

```diff
@@ -92,10 +94,12 @@ cds-tag {
   margin-bottom: 8px;
 }
 
+{% if standalone %}
 p {
   font-size: 14px;
   color: #525252;
 }
+{% endif %}
```

Comment:
I think it'd be helpful to group all the standalone CSS rules together.

Reply:
Good call, I can't move the media rules to the top as it breaks the reactivity, but I've moved the `p` rule.